src/docs/tipsForReusableCode.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/ReusableCode" />
+
+# Tips for writing resuable code
+
+## Context
+
+You need a component to do a slightly different thing than the one you already have but you want to avoid duplicating code. What should you do ? There's no single correct answer. This guide will show you the options and help you appreciate the trade-offs.
```

Comment:
```suggestion
You need a component to do a slightly different thing than the one you already have, but you want to avoid duplicating code. What should you do? There's no single correct answer. This guide will show you the options and help you appreciate the trade-offs.
```